src/test/java/com/google/devtools/build/lib/unix/ProcMeminfoParserTest.java

```diff
@@ -75,6 +76,7 @@ public void memInfo() throws Exception {
     assertThat(memInfo.getFreeRamKb()).isEqualTo(14717640);
     assertThat(memInfo.getRamKb("Cached")).isEqualTo(509940);
     assertThat(memInfo.getTotalKb()).isEqualTo(3091732);
+    assertThat(memInfo.getRamKb("Writeback")).isEqualTo(123);
```

Comment:
The code implements first wins, but this asserts last wins. Did you actually run the test?

Reply:
No, since I where confused by the documentation, 
what are the values I need to use for the `...`?
in this command ```bazel test //src/... //third_party/ijar/...```

Reply:
`bazel test //src/test/java/com/google/devtools/build/lib:unix_test` is probably good enough for local testing in this case. :-)